RVFitter/RVFitter_comparison.py

```diff
@@ -187,7 +188,7 @@ def adjust_table(x, variable, shape_profile, constraint, apply_cutoff=False):
             val = sigfig.round(x[value_var], x[error_var], separation=' \pm ')
         return "$" + val + "$"
 
-    def compare_fit_results_1D(self, variable, fig_and_ax=None):
+    def compare_fit_results_1D(self, variable, fig_and_ax=None, suffix=None):
```

Comment:
Isn't it a `prefix`?

Reply:
Thinking about it now: Shouldn't the `prefix/suffix` not be a class-member so that all plots which are made follow this convention and one can easily relate the plot which has been made to the `prefix`?
But this might take away flexibility that you wish. So I leave it up to you.

Reply:
I prefer to have the flexibility